docs/_templates/page.html_orig

```diff
@@ -1,16 +1,16 @@
 {% extends "!page.html" %}
```

Comment:
Do you want to keep this file in the source tree or was it meant for keeping track of the original state?

Reply:
I dunno. I killed it. easy-peasy,